source/js/addons.js

```diff
@@ -0,0 +1,183 @@
+        }
+    });
+}
+
+function rerenderAddons() {
+    renderAddonsSection(addonsMetadataProvider, "addonsMetadataProvider", "Metadata Sources");
+    renderAddonsSection(addonsGeneric, "addonsGeneric", "Generic");
+    renderAddonsSection(addonsThemeDesktop, "addonsThemeDesktop", "Themes Desktop");
+    renderAddonsSection(addonsThemeFullscreen, "addonsThemeFullscreen", "Themes Fullscreen");
+    renderAddonsSection(addonsGameLibrary, "addonsGameLibrary", "Libraries");
+}
+
+function sanitize(input) {
+    var output = input.replace(/<script[^>]*?>.*?<\/script>/gi, '').
+        replace(/<[\/\!]*?[^<>]*?>/gi, '').
+        replace(/<style[^>]*?>.*?<\/style>/gi, '').
+        replace(/<![\s\S]*?--[ \t\n\r]*>/gi, '').
+        replace(/&nbsp;/g, '');
+    return output;
+}
+
+function renderAddonsSection(data, type, name) {
+    var htmlItems = [];
+    if (data.length > 0) {
+        $.each(data, function (_key, val) {
+            htmlItems.push(" \
+            <li class='list-group-item'> \
+                <div class='row'> \
+                    <div class='col-sm-2'> \
+                        " + (val.iconUrl != undefined ? "<img src=" + encodeURI(val.iconUrl) + " width=120 loading='lazy'></img>" : "") + "\
+                    </div> \
+                    <div class='col-sm-9'> \
+                        <h4>" + sanitize(val.name) + "</h4> \
+                        <p class='addon-description'>" + sanitize(val.description == undefined ? val.shortDescription : val.description) + "</p> \
+                " + (val.screenshots != undefined ? "<p> \
+                " + $.map(val.screenshots,
+                function (key, _val) {
+                    return '<a href="' + encodeURI(key.image) + '" target="_blank"><img class="addon-screenshot" src="' + encodeURI(key.thumbnail) + '" loading="lazy"></img></a>';
+                }).join('') + "</p>" : '') + " \
+                <p> \
+                <a href=" + encodeURI(val.sourceUrl) + " target='_blank'>Source code</a> \
+                " + (val.links != undefined ? " | \
+                " + $.map(val.links,
+                    function (link_key, link_val) {
+                        if (link_key !== val.sourceUrl) {
+                            return '<a href="' + encodeURI(link_key) + '" target="_blank">' + sanitize(link_val) + '</a>';
+                        }
+                        else {
+                            return null;
+                        }
+                    }).join(' | ') : '') + " \
+                </p > \
+                <p> \
+                Author: " + sanitize(val.author) + " \
+                </p > \
+                </div > \
+                <div class='col-sm-1'> \
+                <p class='pull-right'> \
+                <a href='playnite://playnite/installaddon/" + encodeURI(val.addonId) + "' class='btn btn-default'>Download</a> \
+                </p> \
+                    </div > \
+                </div > \
+            </li > "
```

Comment:
`encodeURI()` [does not encode the `'` character](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI#description), and this is used inside a single quoted HTML attribute.

Switch to double quoted attributes:

```suggestion
                Author: " + sanitize(val.author) + ' \
                </p> \
                </div> \
                <div class="col-sm-1"> \
                <p class="pull-right"> \
                <a href="playnite://playnite/installaddon/' + encodeURI(val.addonId) + '" class="btn btn-default">Download</a> \
                </p> \
                    </div > \
                </div > \
            </li > '
```